systest/backup/nfs-backup/backup_test.go

```diff
@@ -49,6 +50,13 @@ var (
 )
 
 func TestBackupHAClust(t *testing.T) {
+	// check health of alpha1
```

Comment:
add a comment here explaining the logic, with a `TODO` flag. For example: 
```
//TODO: Polling for health-checks with a 1sec wait for 100 times ... this can be changed to a push event action instead of polling this way. 

We can come back and fix it later with this TODO flag.

Reply:
done @skrdgraph